parm/config/gfs/config.ice

```diff
@@ -6,4 +6,9 @@ echo "BEGIN: config.ice"
 export min_seaice="1.0e-6"
 export use_cice_alb=".true."
 
+export MESH_ICE="mesh.mx${ICERES}.nc"
```

Comment:
Since we are changing names anyway, can we make this `CICE_MESH` to be consistent with the others?

Reply:
`MESH_ICE` is a template variable in `ufs.configure` so we cannot change this variable name.